src/runner.py

```diff
@@ -38,12 +38,12 @@ def __init__(self, configs, args):
         self._verbose = args.verbose
         self._job_tmp_dirs = {}
 
-    def _create_node(self, checkout_node, plan_config):
+    def _create_node(self, create_tarball_node, plan_config):
```

Comment:
How about just `tarball_node`? It's a node about the tarball, first it's created by `tarball.py` but then it's just used by the `runner.py`.

Reply:
Yes, need to rename it.